src/manifolds/Rotations.jl

```diff
@@ -344,6 +344,33 @@ function Random.rand!(
     return pX
 end
 
+"""
+    _get_expdhalf(::Rotations, d)
+
+Calculate `exp(d / 2)`.
+"""
+_get_expdhalf(::Rotations, d) = exp(d / 2)
+"""
+    _get_expdhalf(::Rotations{TypeParameter{Tuple{3}}}, d)
+
+Calculate `exp(d / 2)` for the manifold of `Rotations(3)` based on the Rodrigues' rotation
+formula.
```

Comment:
If it is not too much effort, maybe documenting the formula that is used would be cool?

Reply:
It's just temporary measure until someone has time to derive efficient formula for the entire PT. It's very standard code so nothing is gained by elaborating on it.

Reply:
Hm, well I would gain that I maybe understand it, but I do see that there is no black/white border on what is “easy enough”. I had colleagues that claimed that Matlab is high-level enough that one never needs documentation...

But ok, then maybe document the entire PT variant more detailed once it is available.
I know that these formulae here are also used somewhere else and they come from a time before we documented formulae and their origin, I would like to slowly get rid of undocumented formulae.

Reply:
This is not a time for handling that issue. This is not a new complicated formula.